i18n/en/package.i18n.json

```diff
@@ -1,5 +1,5 @@
 {
-	"espIdf.createFiles.title": "Create ESP-IDF project",
+	"espIdf.createFiles.title": "ESP-IDF: Create ESP-IDF project",
```

Comment:
I think we could extend scope of this PR and fix one more minor issue with titles:

Some commands have `.` in the end of the title, while others don't, for example

```
"espIdf.selectPort.title": "ESP-IDF: Select port to use.",
```

I think it's better to get rid of all these `.`

Reply:
One more thing, is n't it clear enough: `ESP-IDF: Create  project`?